packages/quantic/force-app/main/default/lwc/quanticSmartSnippetSuggestions/quanticSmartSnippetSuggestions.css

```diff
@@ -1,3 +1,7 @@
 .smart-snippet-suggestions__title {
   font-weight: 700;
 }
+
+.accordion__summary-heading {
+  --slds-c-accordion-heading-font-size: 1.1rem;
```

Comment:
I'm wondering if instead of overriding this to a `1.1rem` value, we could instead set a css class on the container to influence the inner font-size?

For example by adding `class="slds-text-body_small"` maybe to the `lightning-accordion-section` maybe the content of it would be more aligned with the quantic text size?

Reply:
yes I did think of using the slds class on the lightning-accordion-section, but it doesnt target the header of the section, just the inside content text. The current solution was the only way of targeting the header I could find.